datastore/mock_database.go

```diff
@@ -484,3 +494,13 @@ func (mdb *ErrorMockDB) CreateDeviceNonce() (DeviceNonce, error) {
 func (mdb *ErrorMockDB) ValidateDeviceNonce(nonce string) error {
 	return errors.New("MOCK error validating a nonce")
 }
+
+// CreateOpenidNonceTable database mock
+func (mdb *ErrorMockDB) CreateOpenidNonceTable() error {
+	return nil
```

Comment:
If a CreateTable fails then there is an issue with the database and the whole service will fail (as it gets run on service restart). Not sure if it's really worth the extra effort.

That makes sense, but it means changes to the snaps as well :)